File: library/src/main/java/cc/shinichi/library/view/listener/OnBigImageLongClickListener.java
package cc.shinichi.library.view.listener;

import android.app.Activity;
import android.view.View;

/**
 * @author 工藤
 * @email 18883840501@163.com
 * cc.shinichi.library.view.listener
 * create at 2018/12/19  16:24
 * description:
 */
public interface OnBigImageLongClickListener {

    /**
     * 长按事件
     */
    boolean onLongClick(Activity activity, View view, int position);
}

File: library/src/main/java/cc/shinichi/library/view/listener/OnBigImageLongClickListener.kt
package cc.shinichi.library.view.listener

import android.app.Activity
import android.view.View

/**
 * @author 工藤
 * @email qinglingou@gmail.com
 * cc.shinichi.library.view.listener
 * create at 2018/12/19  16:24
 * description:
 */
interface OnBigImageLongClickListener {
    /**
     * 长按事件
     */
    fun onLongClick(activity: Activity?, view: View?, position: Int): Boolean
}